Scripts/AnimationScripts/PlayerPushAnimation.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerPushAnimation : MonoBehaviour
+{
+    [SerializeField]
+    private Animator myAnimationController;
```

Comment:
My yazmaya gerek yok, animator yazmak daha kısa ve temiz olur :)

```suggestion
    private Animator animator;
```